### ISSUE TITLE
[MM-19945] Add aria-label to Start Meeting button

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-plugin-zoom#59]
#### Summary

To improve accessibility, an `aria-label` should be applied to the Start Zoom Meeting button in the app's header. An acceptable label would be `Camera Icon`.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-19945

#### Screenshots

![image](https://user-images.githubusercontent.com/6913320/71053530-ab012000-211c-11ea-88a4-cf8dbb7eda61.png)

---
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/) and our [Plugins Guide](https://developers.mattermost.com/extend/plugins/).
